UPDATING.md

```diff
@@ -1720,6 +1727,37 @@ support for SSL connection to HDFS).
 
 SSL support still works for WebHDFS hook.
 
+### Unify user session lifetime configuration
```

Comment:
This is also under `Master` section. We should remove it from `Master` section at the same time.

Reply:
Right, yes. I though something was missing :D. Now I know what was -- "the deletions"

Reply:
Updated in https://github.com/apache/airflow/pull/12624/commits/b9a908f871d2b72616d35bc545f48c2a36deeb4f